INSTALL

```diff
@@ -3,18 +3,4 @@
 
 The easiest way to install is via opam:
 
-    opam init
-    opam remote add xen-org git://github.com/xen-org/opam-repo-dev
```

Comment:
Should we link to https://github.com/xapi-project/xs-opam#adding-the-opam-repository?